corehq/apps/accounting/models.py

```diff
@@ -459,7 +459,6 @@ def get_or_create_account_by_domain(cls, domain,
         ), True
 
     @classmethod
-    @quickcache(['domain'])
```

Comment:
nit: these should probably be two separate commits (text color / revert caching). You can use `git add -p` to select which local changes you want to add to a commit.  That's mostly just a tip going forward; this PR doesn't need to block on cleaning up the commit history or anything